textanalytics/low-level-dynamic/azure-ai-textanalytics/README.md

```diff
@@ -0,0 +1,186 @@
+# Azure REST Bindings for Azure Cognitive Services Text Analytics client library for Java
+
+azure-ai-textanalytics-protocol is an experimental library for interacting directly with the REST endpoints for the Azure Cognitive Services Text Analytics service.
+
+At this time, if you are looking to interact with the Azure Cognitive Services Text Analytics service, we recommend using the officially supported [Azure.AI.TextAnalytics](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/textanalytics/azure-ai-textanalytics) library.
+
+## Getting started
+
+### Install the package
+
+### Prerequisites
+
+### Authenticate the client
+
+To create the client, you can use an `AzureKeyCredential`, with your API Key:
+
+```java
+TextAnalyticsClient client = new TextAnalyticsClientBuilder()
+        .credential(new AzureKeyCredential("{api key from portal}"))
```

Comment:
For the purpose of this user study, I think we should short-circuit this and store the API key in env var and read from there. Users spend quite a bit of time trying to locate the api key on the portal.

Reply:
Yes that's what the user study is doing: https://github.com/jianghaolu/low-level-java-proto/pull/2/files#diff-a1d4bdc1088c7eb613ef35ed82667215e643dffdd3054c7693ec54da0ab5523fR9